Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/MousePointer.cs

```diff
@@ -75,15 +82,27 @@ public override IMixedRealityController Controller
         /// <inheritdoc />
         public override void OnPreSceneQuery()
         {
+            // screenspace to ray conversion
             transform.position = CameraCache.Main.transform.position;
 
-            Ray ray = new Ray(Position, Rotation * Vector3.forward);
+            Ray ray = new Ray(transform.position, transform.forward);
```

Comment:
>Ray ray = new Ray(transform.position, transform.forward); [](start = 12, length = 57)

As far as I understand, the idea for pointers derived from controller pose synchronizer is that they will take the origin and direction from the pose reported by the controller. In that sense, here we should be using directly Position and Rotation * Vector3.forward for the ray.